coreos-base/oem-packet/files/cloud-config.yml

```diff
@@ -0,0 +1,41 @@
+#cloud-config
+
+coreos:
+  units:
+    - name: oem-reset-interfaces.service
```

Comment:
Should this run after the oem-cloudinit service? Otherwise, it will reconfigure with the original configs.